### ISSUE TITLE
module is not compatible with puppet 6.5.0

### ISSUE THREAD

**Issue by vchepkov** [puppetlabs/puppetlabs-puppetserver_gem#29]
get the following error:

```
Jun 19 19:19:45 master puppet-agent[7559]: Could not prefetch package provider 'puppetserver_gem': No command gemcmd defined for provider puppetserver_gem
Jun 19 19:19:45 master puppet-agent[7559]: Failed to apply catalog: No command gemcmd defined for provider puppetserver_gem
Jun 19 19:19:46 master puppet: /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:98:in `exit': no implicit conversion from nil to integer (TypeError)
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:98:in `block in run_in_fork'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:94:in `fork'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:94:in `run_in_fork'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:48:in `block in run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application.rb:179:in `controlled_run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/agent.rb:46:in `run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/daemon.rb:170:in `block in run_event_loop'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/scheduler/job.rb:49:in `run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/scheduler/scheduler.rb:39:in `block in run_ready'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/scheduler/scheduler.rb:34:in `each'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/scheduler/scheduler.rb:34:in `run_ready'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/scheduler/scheduler.rb:11:in `run_loop'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/daemon.rb:193:in `run_event_loop'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/daemon.rb:149:in `start'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/agent.rb:389:in `main'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/agent.rb:354:in `block in run_command'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/context.rb:65:in `override'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet.rb:266:in `override'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application/agent.rb:350:in `run_command'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application.rb:382:in `block in run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util.rb:684:in `exit_on_fail'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/application.rb:382:in `run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/command_line.rb:139:in `run'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet/util/command_line.rb:77:in `execute'
Jun 19 19:19:46 master puppet: from /opt/puppetlabs/puppet/bin/puppet:5:in `<main>'
```

**Comment by simmerz** [comment]
Looks like this has been fixed in master and documented here: https://tickets.puppetlabs.com/browse/PUP-6488

**Comment by vchepkov** [comment]
It does, thanks to @tkishel , but no forge module yet

**Comment by vchepkov** [comment]
fixed in version 1.1.1
